Scripts/Proxies/ProxyAnimator.cs

```diff
@@ -8,18 +9,34 @@
 [RequireComponent(typeof(ProxyHelper))]
 public class ProxyAnimator : MonoBehaviour, ICustomActionMap
 {
-	class TransformInfo
+	public class TransformInfo
 	{
 		public Vector3 initialPosition;
 		public Vector3 initialRotation;
 		public Vector3 positionOffset;
 		public Vector3 rotationOffset;
 
-		public void Reset()
+		public void ResetOffsets()
 		{
 			positionOffset = Vector3.zero;
 			rotationOffset = Vector3.zero;
 		}
+
+		public void ResetPositionOffset()
```

Comment:
Just make the position and rotation offsets properties and set them to zero directly?  These methods seem redundant

Reply:
Yeah, they're already public so looking back I'm not sure why I didn't do this originally